vnext/Shared/OInstance.cpp

```diff
@@ -151,6 +152,9 @@ std::string GetJSBundleFilePath(const std::string &jsBundleBasePath, const std::
 } // namespace
 
 using namespace facebook;
+using namespace Microsoft::JSI;
+
+using std::make_shared;
```

Comment:
(nit) why is this needed?

Reply:
Reducing explicit namespace usage and thus line lengths.